unblob/handlers/executable/elf.py

```diff
@@ -43,6 +43,7 @@ def extract(self, inpath: Path, outdir: Path):
         is_kernel = (
             elf.header.file_type == lief.ELF.E_TYPE.EXECUTABLE
             and elf.has_section(KERNEL_SYMBOLS_SECTION)
+            # TODO: this is incomplete: __ksymtab is only present if "loadable module support" is enabled!!!
```

Comment:
Why look up for `__ksymtab` (KERNEL_SYMBOLS_SECTION) when we could just look for `.init.data` (KERNEL_INIT_DATA_SECTION) ? This way we also cover cases when "loadable module support" is not enabled, and we can get rid of a superfluous if on line 63 if we do the check here.

Reply:
good catch! changed.